blocks/landing-page-hero/landing-page-hero.js

```diff
@@ -158,35 +158,37 @@ async function fetchFormThanksElem(thanksUrl) {
 async function decorateCtasContent(ctasContent, stylesList) {
   // Only allows one form to be linked from a landing-page-hero, if a need should arise to allow multiple will need to adjust this implementation
   const formButton = ctasContent.querySelector('a[href*="forms/"]');
-  const rawLink = formButton.getAttribute('href');
-
-  let styleClass;
-  // eslint-disable-next-line no-restricted-syntax
-  for (const style of stylesList) {
-    const styleClassTmp = overlayStyles.find((o) => o.name === style);
-    if (styleClassTmp !== undefined) {
-      styleClass = styleClassTmp.class;
-      break;
+  if (formButton) {
+    const rawLink = formButton.getAttribute('href');
+
+    let styleClass;
+    // eslint-disable-next-line no-restricted-syntax
+    for (const style of stylesList) {
+      const styleClassTmp = overlayStyles.find((o) => o.name === style);
+      if (styleClassTmp !== undefined) {
+        styleClass = styleClassTmp.class;
+        break;
+      }
     }
-  }
 
-  formButton.replaceWith(getFormSection(formButton, styleClass));
+    formButton.replaceWith(getFormSection(formButton, styleClass));
 
-  const formPath = rawLink.substring(rawLink.indexOf('/') + 1);
-  const formWrapper = ctasContent.querySelector(overlayStyles.map((overlayStyle) => `.${overlayStyle.class}`).join(','));
-  const formContent = fetchFormContent(`${formPath}`, formWrapper);
-  formWrapper.append(createFormOverlay(await formContent));
+    const formPath = rawLink.substring(rawLink.indexOf('/') + 1);
```

Comment:
This is not as clear or reliable as using the URL object.
const formPath = new URL(rawLink.href).pathname;